ReSharper.FSharp/test/data/features/generate/overrides/Anchor - No repr 02.fs.gold

```diff
@@ -9,3 +9,4 @@
 
 type T() =
   override this.ToString() = {selstart}failwith "todo"{selend}
+    
```

Comment:
@dawedawe Could you please check why this was changed?

Reply:
That's resolved by using `ITreeNode.Contains` :)